UI/cypress/e2e/shortlist.js

```diff
@@ -144,3 +143,39 @@ When(
     cy.wait(500);
   }
 );
+
+Then("total amount of shortlisted TPs is {int}", (expectedTotal) => {
+  cy.checkTotalTps(expectedTotal);
```

Comment:
This is quite a specific thing, but with a generic name, on the global object. Not sure we need any of these on the global object any more do we?